app/src/main/res/values/strings.xml

```diff
@@ -630,6 +630,7 @@
     <string name="TermsOfUse" translatable="false">Terms and Privacy</string>
     <string name="terms_url">https://world.openfoodfacts.org/terms-of-use</string>
     <string name="faq_url" translatable="false">https://world.openfoodfacts.org/faq</string>
+    <string name="ingredient_url" translatable="false">https://world.openbeautyfacts.org/ingredient</string>
```

Comment:
Can you rely on the api base url instead ? That way, it can work across projects and languages in the future.

Reply:
yeah okay.

Reply:
actually, you can use the string called "website"

Reply:
it will be https://world-fr.openfoodfacts.org in Open Food Facts, https://world-fr.openbeautyfacts.org and it changes according to language, so you don't have anything to worry about.

Reply:
ok I will use it then.